desktop/src/main/java/bisq/desktop/bisq.css

```diff
@@ -804,6 +804,10 @@ bg color of non edit textFields: fafafa
     -fx-fill: -bs-rd-nav-deselected;
 }
 
+.nav-button-japanese .text {
```

Comment:
Why is this change part of this PR? I think something got wrong during rebasing of your branch.